cmd/helm2go-operator-sdk/new/cmd_test.go

```diff
@@ -138,6 +169,12 @@ func createValidCommand() *cobra.Command {
 	return cmd
 }
 
+func cleanupValidOperator() {
+	gopathDir := os.Getenv("GOPATH")
+	operatorDir := filepath.Join(gopathDir, "src", "redhat-nfvpe", "helm2go-operator-sdk", "cmd", "helm2go-operator-sdk", "new", "test-operator")
```

Comment:
move "redhat-nfvpe" repo related to a constant so easy to refactor when moved to different repo